web/src/test/java/org/openmrs/web/WebUtilTest.java

```diff
@@ -21,6 +22,52 @@
  */
 public class WebUtilTest {
 	
+
+	/*
+	 * Add to achieve clause coverage of predicate (c<=0x20 ||... in normalizedLocal method. 
+	 * 	C1: c <= 0x20
+	 * 	C2: c >= 0x7f
+	 * 	C3: c >= 0x20
+	 * 	C4: c <= 0x7f
+	 * 	C5: Character.isLetter(c)
+	 * 	C6: c != 0x5f
+	 */
+	 
+	@Test
+	public void normalizeLocalC1T_C2F_C3F_C4T_C5F(){
+		byte[] bytes = {0x11};
+		try{
+		    String s = new String(bytes, "ASCII");
+		    Assert.assertEquals(null, WebUtil.normalizeLocale(s));
+		}
+		catch(IOException ioe){
+		
```

Comment:
After catching this exception you should just throw the exception, it's unlikely the desired behavior is to do nothing in light of a IOException